subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -1170,9 +1189,9 @@ static void ticker_op_cb(uint32_t status, void *param)
 }
 #endif /* CONFIG_BT_CTLR_CENTRAL_SPACING  == 0 */
 
-static void set_bn_max_pdu(bool framed, uint32_t iso_interval,
-			   uint32_t sdu_interval, uint16_t max_sdu, uint8_t *bn,
-			   uint8_t *max_pdu)
+static uint32_t set_bn_max_pdu(bool framed, uint32_t iso_interval,
+			       uint32_t iso_interval_cig, uint32_t sdu_interval,
+			       uint16_t max_sdu, uint8_t *bn, uint8_t *max_pdu)
```

Comment:
Not sure if I agree that the naming of this function implies that it returns the iso interval in microseconds. 

Suggest a better name and/or documentation.